vistrails/packages/vtDV3D/API.py

```diff
@@ -292,8 +292,8 @@ def run(self):
         gui.application.stop_application()        
         
 if __name__ == '__main__':
+    import cdutil, genutil
     cdmsfile = cdms2.open('~/data/AConaty/comp-ECMWF/ecmwf.xml')
```

Comment:
This is debugging code embedded in obsolete code.   There should not be a need to review this code- it's not used anymore.

From: Charles Doutriaux <notifications@github.com<mailto:notifications@github.com>>
Reply-To: UV-CDAT/VisTrails <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, February 24, 2015 4:50 PM
To: UV-CDAT/VisTrails <VisTrails@noreply.github.com<mailto:VisTrails@noreply.github.com>>
Cc: "Maxwell, Thomas P. (GSFC-606.2)[SCIENCE APPLICATIONS INTL CORP]" <thomas.maxwell@nasa.gov<mailto:thomas.maxwell@nasa.gov>>
Subject: Re: [VisTrails] DV3D fixes (#17)


In vistrails/packages/vtDV3D/API.py<https://github.com/UV-CDAT/VisTrails/pull/17#discussion_r25296354>:

> @@ -292,8 +292,8 @@ def run(self):
>          gui.application.stop_application()
>
>  if __name__ == '__main__':
> +    import cdutil, genutil
>      cdmsfile = cdms2.open('~/data/AConaty/comp-ECMWF/ecmwf.xml')


hard coded path?

—
Reply to this email directly or view it on GitHub<https://github.com/UV-CDAT/VisTrails/pull/17/files#r25296354>.